tests/test_mims.py

```diff
@@ -1,7 +1,19 @@
-from openformat.mims import load_mims
+from openformat.mims import load_mims, remove_correlated_noise, infer_translations, \
+    apply_translations
+
+FILENAME = 'tests/data/2018_04_25_Cd_AF33_1_ROI3.im'
 
 
 def test_load_mims():
-    mims = load_mims('tests/data/2018_04_25_Cd_AF33_1_ROI3.im')
+    mims = load_mims(FILENAME)
     assert len(mims.tab_mass) == mims.mask_im.nb_mass
     assert mims.data.shape[1] == mims.mask_im.nb_mass
+
+
+def test_filter_align():
+    padding = 8
+    mims = load_mims(FILENAME)
+    cleaned = remove_correlated_noise(mims.data)
```

Comment:
@mburkittgray, here's an example of how you might remove hot pixels and align the images using a convolution.